docs/2_4_common_schema.adoc

```diff
@@ -855,1491 +807,1469 @@ Element `<fmiModelDescription><DefaultExperiment>` is defined as:
+|====
 
-_The signal flow can cause algebraic loops._
-_If variables in connected terminals have the <<causality>> <<output>>, then an importing tool may iterate an undefined <<input>> of an FMU to ensure that the connected output values are equal._
+_Discussion of the combinations that are allowed_:
 
-_Another use case is the parameter propagation._
-_If a variable in both connected terminals has the <<causality>> <<parameter>>, then an importing tool could ask the user for the value of one of those <<parameter,`parameters`>> only, and propagate this value to the other FMU._
-_If only one of the variables has <<causality>> <<parameter>>, and the other is a <<constant>> <<output>> or <<calculatedParameter>>, then the importing tool could also propagate the <<parameter>> value without presenting a parameter to the user._
-_One example of use would be the name of a substance flowing through a pipe._
-_If the fluid flows from one pipe FMU to another, the substance should be the same._
-_This substance name could be propagated over several FMUs._
+.Combinations of variability and causality that are allowed and why.
+[[table-Combinations-allowed]]
+[cols="1,3,8", options="header"]
+|====
+|
+|_Setting_
+|_Example_
 
-_Finally the `variableKind` `signal` can be applied to implement compatibility checks._
-_If for example the <<variability>> of the variables in connected terminals are <<constant>>, then the importing tool can implement an equality assertion._
-_This is also possible with <<calculated>> <<parameter,`parameters`>>._
-_One example of use would be the cross sectional flow area in pipes which is calculated from geometry parameters._
-_A change in the cross sectional flow area is relevant for the momentum equation, and therefore the connection has to be deemed incompatible if these variables are present and unequal.]_
+>|_(1)_
+|_<<fixed>> <<parameter>>_
+|_Non-<<tunable>> <<parameter>>_
 
-====== General Remark on Inflow and Outflow
+>|_(2)_
+|_<<tunable>> <<parameter>>_
+|_<<tunable,`Tunable`>> <<parameter>> (changing such a <<parameter>> triggers event handling (ME) or takes effect at the next communication point (CS and SE), and <<tunable>> <<calculatedParameter>>/<<output>>/<<local>> variables might change their values)._
 
-_[Flow variables have a direction and must fulfill a zero sum constraint i.e. the sum of all flow variables connected must be zero (Kirchhoff's current law)._
-_In addition because different tools might have different direction definitions both, `inflow` and `outflow` are available as `variableKind`._
-_For variables with `inflow` a positive value means that the flow is inwards, and for `outflow` a positive value means that the flow is outwards._
-_For the sake of simplicity in the following latexmath:[\dot{m}_i] denotes an inflowing quantity:_
+>|_(3)_
+|_<<fixed>> <<dependenciesKind,`dependent`>> <<parameter>>_
```

Comment:
Shouldn't calculatedParameter be written here instead of dependent parameter?

Reply:
yes, I think so

Reply:
@KarlWernersson , @TorstenBlochwitz , Modelicans: the causality `calculatedParameter` and the dependencyKind `dependent` - are they the same? If so, should we then not rename `calculatedParameter` to `dependentParameter` to be more consistent?

Reply:
@andreas-junghanns , No they refer to different things, calculated parameter is a causality, dependencyKind is for the dependency in model structure, where you have ("constant", "fixed", "tunable") (these are all different types of linear dependency) and dependent which is all other types of dependencies nonlinear, unknown etc. 

However what @IZacharias  is refereeing to is table 19. where we list allowed combinations of casualty and variability, , both (3) **and** (4) refers to dependent parameters, which doesn't exits, it should just be,  fixed, calucatedparameter and tunalbe, calculatedparmaeter .
as it refers to those combinations in table 17.

Reply:
Fixed in the table (now 23).